tests/functions/test_remove_empty.py

```diff
@@ -21,11 +21,24 @@ def test_remove_empty(df):
 
 @pytest.mark.functions
 def test_index_after_remove_empty():
-    # This test ensures that the indexed is reset correctly.
+    """This test ensures that the indexed is reset correctly"""
```

Comment:
Just requesting a small typo fix in the test docstring.
```suggestion
    """This test ensures that the index is reset correctly"""
```